sirmordred/task_enrich.py

```diff
@@ -223,6 +237,47 @@ def __autorefresh(self):
         else:
             logger.debug("No ids to be refreshed found")
 
+        # Update corresponding autorefresh date
+        if studies:
+            self.last_autorefresh_studies = next_autorefresh
+        else:
+            self.last_autorefresh = next_autorefresh
+
+    def __autorefresh_studies(self, cfg):
+        """Execute autorefresh for areas of code study if configured"""
+
+        if 'studies' not in self.conf[self.backend_section] or \
+                'enrich_areas_of_code:git' not in self.conf[self.backend_section]['studies']:
+            logger.debug("Not doing autorefresh for studies, Areas of Code study is not active.")
+            return
+
+        aoc_index = self.conf['enrich_areas_of_code:git'].get('out_index')
+
+        if not aoc_index:
```

Comment:
This will raise an exception in case of `out_index` key won't exist. Is that what we want?

Reply:
It would return `None` in that case. Anyway I've modified the code to return default value if key does not exist. 

Then I check `aoc_index` in case it is still `None`, which means `out_index` exists but has no value. In that case we should also use default value.